docker/stable/Dockerfile.gpu

```diff
@@ -34,7 +34,7 @@ RUN pip install --no-cache-dir \
     ipython \
     ipykernel \
     ipywidgets \
-    papermill \ 
+    papermill \
```

Comment:
don't the stable docker files also need to change the fe version to specify 1.5.1?

Reply:
nice catch, updated.